bin/heapdump.cmd

```diff
@@ -20,5 +20,5 @@ rem   Ask the JMeter client to perform a HeapDump
 
 rem   P1 = command port for JMeter instance (defaults to 4445)
 
-java -cp %~dp0ApacheJMeter.jar org.apache.jmeter.util.ShutdownClient HeapDump %*
+java -cp "%~dp0"ApacheJMeter.jar org.apache.jmeter.util.ShutdownClient HeapDump %*
```

Comment:
In my tests, that didn't work. I had to place the closing quote after `ApacheJMeter.jar` resulting in
`java -cp "%~dp0ApacheJMeter.jar" org.apache.util.ShutdownClient HeapDump %*`
On what version of Windows did you do your tests?

Reply:
You're right. My mistake, sorry. (Mismatching commit and test).
Thanks for double checking!